frontend/src/MessageBoard/components/ActivityTimeline.js

```diff
@@ -55,30 +57,35 @@ export default class ActivityTimeline extends React.Component {
 
 									//only add to the allTheThings array if the end of the messages
 									//array has not been reached
-									if (!loaded)
-										allTheThings = allTheThings.concat(
-											findMsgCommentsByMessage
-										);
+									if (!loaded) {
+										for (var comment of findMsgCommentsByMessage) {
```

Comment:
Unless you have a very convincing argument for using `var` here, please update it to `let`.